run_experiments.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import warnings
+import yaml
+
+from misc.utils import set_seed
+from runners import main_runner
+
+
+warnings.filterwarnings(action="ignore", category=FutureWarning)
+
+
+def parse_args(*args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--config",
+        help="Experiment config file (default: config.yaml).",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "--runner",
+        help="A runner function to execute (default: main_runner).",
+        type=str,
+        default="runner_optimised",
+    )
+    return parser.parse_args(*args)
+
+
+if __name__ == "__main__":
+
+    args = parse_args(
```

Comment:
Keep an eye on that - it will be easy to miss before merging. I suggest to use those arguments via cli when testing and debugging instead of hard-coding them.

Reply:
Ok, I commented that out and provided a comment